i18n.ts

```diff
@@ -17,6 +18,7 @@ export const APP_LANGUAGES = [
   { label: "Français", value: "fr" },
   { label: "Nederlands", value: "nl" },
   { label: "Svenska", value: "sv" },
+  { label: "Italiano", value: "it" },
   { label: "繁體中文", value: "zh-TW" },
```

Comment:
Maybe keep it sorted alphabetically?
```suggestion
  { label: "Français", value: "fr" },
  { label: "Italiano", value: "it" },
  { label: "Nederlands", value: "nl" },
  { label: "Svenska", value: "sv" },
  { label: "繁體中文", value: "zh-TW" },
```